src/RpcClient/ContractClient.cs

```diff
@@ -57,7 +57,7 @@ public async Task<Transaction> CreateDeployContractTxAsync(byte[] contractScript
             byte[] script;
             using (ScriptBuilder sb = new ScriptBuilder())
             {
-                sb.EmitSysCall(ApplicationEngine.System_Contract_Create, contractScript, manifest.ToString());
+                sb.EmitSysCall(ApplicationEngine.System_Contract_CreateStandardAccount, contractScript, manifest.ToString());
```

Comment:
Rename contractScript to nefFile?

Reply:
It's not `System.Contract.CreateStandardAccount`

Reply:
Fixed
```c#
sb.EmitSysCall(ApplicationEngine.System_Contract_Call, NativeContract.Management.Hash, "deploy", new byte[1], manifest.ToString());
```